README.md

```diff
@@ -34,6 +34,12 @@ $result = $client->completions()->create([
 echo $result['choices'][0]['text']; // an open-source, widely-used, server-side scripting language.
 ```
 
+You can also instantiate the client with a custom base api URI:
+
+```php
+$client = OpenAI::clientWithBaseUri('YOUR_API_KEY', 'CUSTOM_BASE_URI');
```

Comment:
Actually, after giving some thoughts into this. I think you should do something different:

```php
OpenAI::factory($apiKey) // returns factory
    ->withBaseUri('string') // returns factory
    ->make(); // makes client.
```

Reply:
I've implemented it that way. Please take a look, again :)